modules/ob-configuring-metric-labels.adoc

```diff
@@ -0,0 +1,49 @@
+// This module is included in the following assembly:
+//
+// builds/build-controller-observability.adoc
+
+:_content-type: CONCEPT
+[id="ob-configuring-metric-labels_{context}"]
+= Configuring a metric label
+
+ You can enable a metric label that you want to use by configuring the `PROMETHEUS_ENABLED_LABELS` environment variable. The following metric labels are supported:
+
+* `buildstrategy`
+* `namespace`
+* `build`
+* `buildrun`
+
+When deploying the build controller on the {ocp-product-title} cluster, you must add the environment variable in the `controller.yaml` deployment file. You can also enable mutiple metric labels by using a comma-separated list of values. 
+ 
+.Prerequisites
+
+*
```

Comment:
@apoorvajagtap, Do we have any prerequisites for this section?

Reply:
@jkhelil need your help here as well.